src/client/components/measures-tile/measures-tile.tsx

```diff
@@ -50,23 +67,28 @@ export class MeasuresTile extends React.Component<MeasuresTileProps, MeasuresTil
     var maxHeight = PIN_TITLE_HEIGHT;
 
     var checkboxType: CheckboxType = multiMeasureMode ? 'check' : 'radio';
-    var rows = dataSource.measures.map(measure => {
+    var rowData = dataSource.measures.toArray();
+    if (searchText) {
+      rowData = rowData.filter((r) => {
+        return r.title.toLowerCase().indexOf(searchText.toLowerCase()) !== -1;
+      });
+    }
+    var rows = rowData.map(measure => {
       var measureName = measure.name;
       var selected = selectedMeasures.has(measureName);
-
       return <div
         className={classNames('row', { selected })}
         key={measureName}
         onClick={this.measureClick.bind(this, measure)}
       >
         <Checkbox type={checkboxType} selected={selected}/>
-        <div className="label">{measure.title}</div>
+        <HighlightString className="label" text={measure.title} highlightText={searchText}/>
       </div>;
     });
-    maxHeight += (rows.size + 2) * MEASURE_HEIGHT + PIN_PADDING_BOTTOM;
 
+    maxHeight += (rows.length + 2) * MEASURE_HEIGHT + PIN_PADDING_BOTTOM;
     const style = {
-      flex: rows.size + 2,
+      flex: rows.length + 2,
```

Comment:
Ok, this wont work well. 'flex' needs to be set to the total number of measures in the datasource. Right now it would jump all over the place if you search.